src/Ormolu/Printer/Internal.hs

```diff
@@ -266,7 +273,7 @@ spit stype text = do
   R $ do
     i <- asks rcIndent
     c <- gets scColumn
-    closestEnclosing <- listToMaybe <$> asks rcEnclosingSpans
+    closestEnclosing <- asks (listToMaybe . rcEnclosingSpans)
```

Comment:
Do you know who made this change, and how is it related to operator chains? It might be a hint suggested by HLS/VSCode that I would have applied by mistake